Include/internal/pycore_ceval.h

```diff
@@ -271,7 +271,7 @@ PyAPI_FUNC(PyObject **) _PyObjectArray_FromStackRefArray(_PyStackRef *input, Py_
 PyAPI_FUNC(void) _PyObjectArray_Free(PyObject **array, PyObject **scratch);
 
 PyAPI_FUNC(PyObject *) _PyEval_GetANext(PyObject *aiter);
-PyAPI_FUNC(PyObject *) _PyEval_LoadGlobal(PyObject *globals, PyObject *builtins, PyObject *name);
+PyAPI_FUNC(void) _PyEval_LoadGlobalStackRef(PyObject *globals, PyObject *builtins, PyObject *name, _PyStackRef *writeto);
```

Comment:
If you create a deferred stack reference, it needs to be stored (visibly) on the interpreter's stack before any escaping calls are made. Within bytecodes.c, this is enforced by the code generator, but external functions need a bit more work.

It's not strictly necessary for `LOAD_GLOBAL` because we don't make any escaping calls after creating the stack reference, but it will be for `LOAD_ATTR` because of the complexity with checking both descriptors and dictionary attributes.

Reply:
What's the motivation for doing this?
(I assume it's performance. Converting from a `PyObject *` counted reference to a deferred stack reference in the interpreter is safe, but slow?)

As I said, `LOAD_GLOBAL` is irrelevant for performance.